modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/portlet/action/GetEntryRenderDataMVCResourceCommand.java

```diff
@@ -936,6 +938,17 @@ private <T extends BaseModel<T>> String _getRender(
 				CTSQLModeThreadLocal.setCTSQLModeWithSafeCloseable(ctSQLMode);
 			UnsyncStringWriter unsyncStringWriter = new UnsyncStringWriter()) {
 
+			ThemeDisplay themeDisplay =
+				(ThemeDisplay)httpServletRequest.getAttribute(
+					WebKeys.THEME_DISPLAY);
+
+			CTConfiguration ctConfiguration =
+				_configurationProvider.getCompanyConfiguration(
+					CTConfiguration.class, themeDisplay.getCompanyId());
+
+			httpServletRequest.setAttribute(
+				"showAllData", ctConfiguration.showAllData());
```

Comment:
Attribute name needs to be added to CTWebKeys.java

Reply:
CTWebKeys is also located at the web module, so I can use it to set the attribute, but not to get it on BaseCTDisplayRenderer